app/views/foreman_leapp/job_templates/leapp_install.erb

```diff
@@ -0,0 +1,36 @@
+<%#
+kind: job_template
+name: Install Leaoo
+job_category: Leapp - Install
+description_format: 'Install leapp package'
+provider_type: Ansible
+feature: leapp_upgrade
+model: JobTemplate
+%>
+<%
+  is_rhel = @host.operatingsystem.name == 'RedHat'
+  os_major = @host.operatingsystem.major.to_i
+-%>
+---
+- hosts: all
+  tasks:
```

Comment:
If the host is not rhel7 or rhel8 then it successfully does nothing. That is a bit misleading